crates/api_common/src/utils.rs

```diff
@@ -360,7 +360,7 @@ pub async fn send_password_reset_email(
   settings: &Settings,
 ) -> Result<(), LemmyError> {
   // Generate a random token
-  let token = generate_random_string();
+  let token = uuid::Uuid::new_v4().to_string();
```

Comment:
This is the only logical change in this PR. I removed `generate_random_string()` and am generating a uuid instead.